### ISSUE TITLE
Python TreeSitter - Error when moving cursor

### ISSUE THREAD

**Issue by JackofSpades707** [LunarVim/LunarVim#4386]
### Problem description

Error message:
```
Error detected while processing CursorMoved Autocommands for "*":
Unable to load context query for python:
...im/0.9.4/share/nvim/runtime/lua/vim/treesitter/query.lua:259: query: invalid structure at position 513 for language python
Press ENTER or type command to continue
```

Has been happening for over a month now. 
I was hoping that after some time passes, updating would fix this issue but it seems that isn't the case.

Issue does not occur for `bash` or `javascript` files.
Bug persists even after Updating/Reinstalling the `python` treesitter parser.

Everything `lvim` is up to date according to the following tools:
`mason`
`lazy`
`TSUpdate`
`LvimUpdate` (Unless this is failing silently.  Nothing happens when running this sometimes, sometimes it does update though, I suspect I only get any UI if an update is available)

### LunarVim version

release-1.3/neovim-0.9-d1c1bace

### Neovim version (>= 0.9.1)

NVIM v0.9.4

### Terminal name

kitty -> tmux -> zsh -> lvim

### Operating system/version

macOS 14.0 (Macbook M2 Pro)

### Steps to reproduce

Reproduce steps:
1. Open python file
2. Move cursor
3. Error occurs once for the process
4. Error no longer occurs during the lifetime of that `lvim` process
5. Treesitter appears to continue working despite the error

Minimal Reproduce env:
- Unknown. Likely just run `TSInstall python` and open a python file and move the cursor.

Full reproduce env:
- https://github.com/JackofSpades707/dotfiles (`lvim` directory)

### Screenshots

![image](https://github.com/LunarVim/LunarVim/assets/17239750/cfa93128-99d2-4950-ae98-74b16f390ad4)

**Comment by antonkesy** [comment]
*TL:DR: treesitter version is (probably) too low*

Maybe try upgrading treesitter manually or switch to nightly to see if the issue persists

might be related to https://github.com/nvim-treesitter/nvim-treesitter-context/issues/298

**Comment by Inovvia** [comment]
For anyone else that experienced this.

> Maybe try upgrading treesitter manually or switch to nightly to see if the issue persists

If you're new to nvim like me and using version 1.3. Here's how I upgraded my treesitter and fixed this error.
1. I went to `lunarvim\site\pack\lazy\opt\nvim-treesitter` (the lunarvim folder will depend on your OS).
2. I switched the branch by opening a terminal in that folder and running `git checkout master`.
3. Then I pull it by using `git pull`.
4. After updating my treesitter I opened LunarVim and ran the following command `:TSUpdate`.

I hope this might help others as treesitter is pinned on lunarvim.
